CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+### Changed
+
+- upgrade `ansible-lint` to `5.3.2`
+
```

Comment:
IMO there is no need to mention development package upgrades in the CHANGELOG. WDYT?

Reply:
I have no opinion.

Reply:
I have.

Reply:
As always

Reply:
Modification made my lord.